public/index.html

```diff
@@ -31,6 +31,7 @@ <h4 id="places">Places</h4>
 <li><a href="http://browserify.org">browserify on the web</a></li>
 <li><a href="https://github.com/maxogden/browser-module-sandbox">browser-module-sandbox</a> - helpful for using multi-bundles</li>
 <li><a href="http://requirebin.com">requirebin.com</a> - powered by browserify-cdn</li>
+<li><a href="http://esnextb.in">esnextb.in</a> - also powered by browserify-cdn</li>
```

Comment:
No, I just realized I haven't regenerated the README in forever.

It looks like marked's behavior has changed too. Curious!